### PR TITLE
[UPGRADE] Add note about api routing change

### LINKED ISSUE #7421: [API] No oauth route
Hello, there is probably a bug (or I miss something), because the ouath route dissapeared.
```
http://localhost:8000/api/oauth/v2/token
```
Response:
```
{
  "code": 404,
  "message": "No route found for \"GET /api/oauth/v2/token\""
}
```

Why has it happened? Is there a posibility to fix that quickly? Because now I cannot use API at all due to lack of authorization.

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | fixes #7421 |
| License         | MIT |

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 ### ApiBundle
 
-* All api routes are prefixed with version number. For example route: `/api/customers` has changed to `/api/v1/customers`. 
+* Change the import path of your API routing from `src/Sylius/Bundle/ApiBundle/Resources/config/routing/main.yml` to `src/Sylius/Bundle/ApiBundle/Resources/config/routing.yml`. From now one you need to add a version prefix to your API calls(for example route: `/api/customers` has changed to `/api/v1/customers`).
```
Comment (pamil) [inline UPGRADE.md]: `calls(for` -> `calls (for` - missing space

Comment (pamil) [inline UPGRADE.md]: `From now one you need to add a version prefix to your API calls(for example route: `/api/customers` has changed to `/api/v1/customers`).`

->

```API became versioned, so you need to prefix them accordingly (e.g. `/api/customer` -> `/api/v1/customer`).```

Wdyt?

Comment (lchrusciel) [inline UPGRADE.md]: nice one :)

**Comment by pjedrzejewski** [comment]
@lchrusciel Requires rebase.

**Comment by lchrusciel** [comment]
@pjedrzejewski done

**Comment by pjedrzejewski** [comment]
👍

**Pull request merged**